src/main/java/edu/cornell/kfs/concur/ConcurUtils.java

```diff
@@ -16,7 +16,7 @@ public static boolean isTravelRequestURI(String URI) {
     }
 
     public static String extractKFSInfoFromConcurString(String concurString) {
-        String kfsInfo = StringUtils.EMPTY;
+        String kfsInfo = concurString;
         if (doesStringContainOpenCloseParenthesis(concurString)) {
             kfsInfo = concurString.substring(concurString.lastIndexOf(CUKFSConstants.LEFT_PARENTHESIS) + 1, concurString.lastIndexOf(CUKFSConstants.RIGHT_PARENTHESIS));
```

Comment:
I am curious about this function.  As written, if "(some bad text (GOOD TEST) some more bad text)" is passed in, the function will return "GOOD TEXT) some more bad text".  I am not sure if you need to be worried about that case or not, but wanted to throw that out here.

I don't think the name of the function says what it is doing.  I am not sure what a concise alternative would be either, so I don't think it needs to be changed right now.